esp/esdllib/esdl_def.cpp

```diff
@@ -1461,6 +1462,8 @@ IEsdlDefObjectIterator* EsdlDefinition::getDependencies( const char* service, St
         methodIter.setown(serviceDef->getMethods());
         for( methodIter->first(); methodIter->isValid(); methodIter->next() )
         {
+            if ((flags & DEPFLAG_ECL_ONLY) && methodIter->query().getPropInt("ecl_hide"))
```

Comment:
perhaps outside of the scope of this issue, but all the available attributes should be well defined (#define, etc), here's a list of other attributes I found:
"logMethod"
"flReqFormat"
"flRespFormat"
"mbsiProductAssociations"
"item_tag"
"request_type"
"response_type"
"product_"
"productdefault_"
"productAssociation"
"ptdselectors"
"selectors"
"translations"
"optional"
"min_ver"
"max_ver"
"min_flver"
"max_flver"
"depr_flver"
"name"
"version"
"base_type"
"enum_type"
"type"
"ecl_hide"
"complex_type"
"variableRecordLengthProcessing"
"allowMultipleEntryPerUnitNumber"